schemas/session-replay/_common-record-schema.json

```diff
@@ -0,0 +1,17 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "_common-record-schema.json",
+    "title": "CommonRecordSchema",
+    "type": "object",
+    "description": "Schema of common properties for a Record event type.",
+    "required": [
+        "timestamp"
+    ],
+    "properties": {
+        "timestamp": {
+            "type": "integer",
+            "description": "Defines the UTC time in milliseconds when this Record was performed.",
+            "readOnly": true
```

Comment:
I assume it should be computed relatively to 00:00:00 UTC on 1 January 1970, but this info is missing in the `description`. Also, we don't expect it to get negative, so we can define `minimum: 0` as we do in [RUM events](https://github.com/DataDog/rum-events-format/blob/b01e4821e5dd59e8fab780eae68329e7b4deabf1/schemas/rum/_common-schema.json#L12). This way we will be able to safely generate `Int64` type for holding this property.